### PR TITLE
feat(ia): Email Sender Field

### PR DESCRIPTION
**Pull request by atligudl**
# Institution Application - Email Sender Field

Attach a link to issue if relevant

## What

contact name and email set as emailFrom when sending application submission emails

## Why

Request from PO

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by eirikurn — APPROVED** [review]

File: libs/application/template-api-modules/src/lib/modules/templates/institution-collaboration/emailGenerators/applicationEmail.ts [hunk 1]
```diff
@@ -44,13 +44,19 @@ export const generateApplicationEmail: ApplicationEmail = (
...
+    | undefined
   const overview = applicationOverviewTemplate(application)
   const body = dedent(`<h2>Yfirlit umsóknar</h2> ${overview}`)
 
   return {
     from: {
-      name: applicationSenderName,
-      address: applicationSenderEmail,
+      name: contactName ?? applicationSenderName,
+      address: contactEmail ?? applicationSenderEmail,
```
Comment (eirikurn) [inline libs/application/template-api-modules/src/lib/modules/templates/institution-collaboration/emailGenerators/applicationEmail.ts:59]: Hm, what's the use case for this? I assume spam filters will block us if we start sending emails from arbitrary email addresses.

**Review by eirikurn — CHANGES_REQUESTED** [review]

File: libs/application/template-api-modules/src/lib/modules/templates/institution-collaboration/emailGenerators/applicationEmail.ts (hunk 1 above)
Comment (atligudl) [reply libs/application/template-api-modules/src/lib/modules/templates/institution-collaboration/emailGenerators/applicationEmail.ts:59]: Good point, I'll raise the issue with our PO

Comment (GunnarIngiReykjalin) [reply libs/application/template-api-modules/src/lib/modules/templates/institution-collaboration/emailGenerators/applicationEmail.ts:59]: The usecase is that there is a ticketing system on the other end (Zendesk) where communication with the applicant takes place. In the current setup, the sender is island@island.is so notifications from the ticketing system never reach the applicant.
That being said, we do indeed risk being flagged as SPAM if we introduce this functionality.
I'm discussing this with our BO

Comment (GunnarIngiReykjalin) [reply libs/application/template-api-modules/src/lib/modules/templates/institution-collaboration/emailGenerators/applicationEmail.ts:59]: Update. Lets scrap this functionality.
We will take a look at using an API with the ticketing system after the summer.

Comment (GunnarIngiReykjalin) [reply libs/application/template-api-modules/src/lib/modules/templates/institution-collaboration/emailGenerators/applicationEmail.ts:59]: @atligudl FYI

**Pull request closed without merging**